docs/devices-api/api-reference/get-nutrition-data.mdx

```diff
@@ -44,72 +41,100 @@ const response = await metriportClient.getNutritionData(
 [
   {
     "metadata": {
-      "date": "2022-12-21",
+      "date": "2023-06-11",
       "source": "cronometer"
     },
     "summary": {
       "macros": {
         "alcohol_g": 0,
-        "carbs_g": 220,
-        "cholesterol_mg": 340,
-        "energy": 2342,
-        "fat_g": 56,
-        "fiber_g": 82,
-        "protein_g": 156,
-        "sodium_mg": 1430,
-        "sugar_g": 12,
-        "trans_fat_g": 0
+        "carbs_g": 129.2,
+        "cholesterol_mg": 460.26,
+        "energy_kcal": 955.4,
+        "fat_g": 24,
+        "fiber_g": 17,
+        "protein_g": 64.1,
+        "sodium_mg": 467.4,
+        "sugar_g": 20.69,
+        "trans_fat_g": 0.46,
+        "water_ml": 595.7
       },
       "micros": {
-        "biotin_mg": 52,
-        "caffeine_mg": 240,
-        "calcium_mg": 564,
-        "copper_mg": 134,
-        "folate_mg": 84,
-        "folic_acid_mg": 12,
-        "iodine_mg": 435,
-        "iron_mg": 232,
-        "magnesium_mg": 4543,
-        "manganese_mg": 198,
-        "phosphorus_mg": 124,
-        "potassium_mg": 128,
-        "selenium_mg": 192,
-        "vitamin_A_mg": 76,
-        "vitamin_B12_mg": 346,
-        "vitamin_B1_mg": 234,
-        "vitamin_B2_mg": 598,
-        "vitamin_B3_mg": 34,
-        "vitamin_B5_mg": 198,
-        "vitamin_B6_mg": 234,
-        "vitamin_C_mg": 654,
-        "vitamin_D_mg": 23,
-        "vitamin_E_mg": 546,
-        "vitamin_K_mg": 432,
-        "zinc_mg": 124
+        "caffeine_mg": 0,
+        "calcium_mg": 297.56,
+        "copper_mg": 0.83,
+        "folate_mg": 379.01,
+        "iron_mg": 17.68,
+        "magnesium_mg": 249.85,
+        "manganese_mg": 4.66,
+        "phosphorus_mg": 895.25,
```

Comment:
if we're going to update this, we need to make sure that we don't miss properties - ie I see at lease `biotin_mg` and `folic_acid_mg` are missing.

Let's double check the updated examples to ensure we're not missing other example properties as well.

Reply:
Done